code/modules/client/client_defines.dm

```diff
@@ -119,3 +119,6 @@
 
 	//Tick when ghost roles are useable again
 	var/next_ghost_role_tick = 0
+
+	// Metacoin balance, cached to reduce DB queries. Do not directly read or modify this value.
+	var/metacoin_balance
```

Comment:
I feel like this should be somehow protected against admin intervention

Reply:
(it is)

Reply:
check the diffs in client_procs.dm ;)